.gitignore

```diff
@@ -63,3 +63,7 @@ __init__.pyi
 
 # Logs
 *.log
+
+# Lightbulb and test bot
+lightbulb/
```

Comment:
why is light bulb in your testing environment? couldn't you just use `pip install -e path_to_local_project`?

Reply:
cuz i was having issues with adding it to the test env